src/Pages/Main/OnlynHot/OnlynHot.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import ItemBox from "../../../Components/Main/ItemBox";
+import "./OnlynHot.scss";
+
+class OnlynHot extends Component {
+  constructor() {
+    super();
+    this.state = {
+      onlyOneItems: [],
+      hotDealItems: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/mockdata/data.json")
+      .then((res) => res.json())
+      .then((res) => {
+        this.setState({
+          onlyOneItems: res.onlyOneItems,
+          hotDealItems: res.hotDealItems,
+        });
```

Comment:
객체의 변수명과 키값이 같을 경우 객체 단축 속성명을 활용해 합쳐줄 수 있습니다.

```suggestion
      .then(({ onlyOneItems, hostDealItems }) => {
        this.setState({ onlyOneItems, hotDealItems });
       }
```